### ISSUE TITLE
Parsing of config.ini file fails if not CR/LF

### ISSUE THREAD

**Issue by MXall0c** [lakiw/compiled-pcfg#1]
The encoding of the ruleset is misread from the config.ini file if the config file is not cr/lf terminated.

Ruleset trained on full rockyou. Python version successfully loads and generates PW candidates.

Attempting to use compiled guesser fails on load.

![image](https://user-images.githubusercontent.com/42156092/74079334-b240fd80-49ea-11ea-8ba5-a685487bcd55.png)

Example Default config.ini file will load & run with compiled version. Comparing generated to provided showed the example to be cr/lf line terminated, generated one was lf terminated (as is usual in linux).

running unix2dos on config file allows it to load.

**Comment by lakiw** [comment]
Good catch, and thank you very much for posting the issue! I'll take a look at it tonight and try to get a fix out in the next couple of days. I probably should also standardize the Python trainer to only use the linux LF terminated to minimize issues like this popping up in the future. So once again thank you. If you notice any other problems, (and I'm sure there are a lot), please don't hesitate to post them here as well. I've been focusing on a different project so I haven't gotten around to giving this project as much attention as it deserves so issue postings like this help refocus me.

**Comment by aaronjones111** [comment]
Glad to hear, thank you!

**Comment by lakiw** [comment]
As a quick update, I haven't forgotten this but I did come down with a cold, and what limited screentime I've had I've been dealing with another security disclosure that will likely occur this week. So once again I really appreciate you highlighting these issues and I wanted you to know they have hit the "feeling guilty about not fixing them by now" stage of my workflow ;p
